simphony/scripts/cuba_generate.py

```diff
@@ -0,0 +1,99 @@
+import click
+import yaml
+
+# Cuba keywords that are excludes from DataContainers
+CUBA_DATA_CONTAINER_EXLCUDE = ['Id', 'Position']
```

Comment:
Good point. We could have an attribute `data_container` that has a true or false value based on the fact that it is one of the DataContainer supported keywords. Other containers could also have their own attribute.

This is related with https://github.com/simphony/simphony-common/issues/29 where we will have groups off keywords which various dictionary like container will support (see discussion in https://github.com/simphony/simphony-lammps-md/pull/6#discussion_r23318337). 

While I believe that you are right and we should implement the new attribute in yaml, I would prefer to implement this improvment in a separate PR, what do you think.